package.json

```diff
@@ -43,7 +43,8 @@
     "hoist-non-react-statics": "^1.0.5",
     "invariant": "^2.2.1",
     "is-promise": "^2.1.0",
-    "lodash": "^4.12.0"
+    "lodash": "^4.12.0",
+    "react-addons-shallow-compare": "^15.0.0"
```

Comment:
@ooflorent I think is more appropriate to add to dependency compare to peer. @erikras  Let me know if I am wrong.